salt/modules/chocolatey.py

```diff
@@ -368,6 +368,7 @@ def install(name,
             override_args=False,
             force_x86=False,
             package_args=None,
+            execution_timeout=None,
```

Comment:
Can you move this new option to the end of the list? We don't want to cause an API change for people here.

Reply:
Done.